smartdashboard/Experiment_Overview.py

```diff
@@ -64,9 +66,10 @@ def build_app(manifest_path: str) -> None:
         )
 
         while True:
+            if manifest_reader.has_changed:
+                st.rerun()
```

Comment:
Chris and I tried to get `watchdog` to work to monitor the user files and streamlit does not play nice with that at all. I'll check about rebuilding the manifest into the `while True:` loop, but I'm 99% sure the app still needs to rerun in order to have the manifest changes show up.

Reply:
Update: streamlit does need to rerun the app. We were trying to use `watchdog` to trigger a rerun of `build_app`, but streamlit behaves really strange when using `watchdog` for user files rather than their own files.